scripts/concurrent_task_utils.py

```diff
@@ -42,6 +44,54 @@ def log(message, show_time=False):
             python_utils.PRINT(message)
 
 
+class TaskResult(python_utils.OBJECT):
+    """Output stream for concurrent_task_utils."""
+
+    def __init__(self, name, failed, error_messages, full_messages):
+        """Constructs a TaskResult object.
+
+        Args:
+            name: str. Name of the check running.
+            failed: bool. Status of the check currently running.
+            error_messages: list(str). List of summary messages returned by
+                the objects.
+            full_messages: list(str). List of full messages returned by the
+                objects.
+        """
+        self.name = name
+        self.failed = failed
+        self.messages = error_messages
+        self.full_messages = full_messages
+
+    @property
+    def all_messages(self):
+        """Returns the full list with the message if the current check is
+        failing or passing.
+
+        Returns:
+            list(str). List of full messages corresponding to currently running
```

Comment:
Currently running task? The task would have been finished right while we print these results?